third_party/sentry-cocoa/Cartfile.resolved

```diff
@@ -1 +1 @@
-github "getsentry/sentry-cocoa" "7.31.2"
+github "getsentry/sentry-cocoa" "7.31.3"
```

Comment:
This is odd. I thought the resolved file would pin the version. If it doesn't, then perhaps we should remove it.

Reply:
That's what I meant, but it seems like it shouldn't be deleted or ignored: https://stackoverflow.com/questions/34688075/safe-to-ignore-carthage-resolved

If it's being changed, it's not working as intended. I think something is wrong with the build.